ci-operator/config/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-master.yaml

```diff
@@ -20,7 +20,7 @@ base_images:
     namespace: edge-infrastructure
     tag: assisted-service
   assisted-service-operator-index:
-    name: ocm-2.7
+    name: ocm-2.6
```

Comment:
I meant to start this PR as WIP but while I have your attention...
@mkowalski and I spend some time yesterday trying to figure out why this job constantly fail to start the infrastracture-operator, we noticed that it's not starting because it can't pull the ci-index image:
```
 message: Back-off pulling image "registry.build04.ci.openshift.org/ci-op-xxi8iphf/pipeline:ci-index"
        reason: ImagePullBackOff
```
Note that this test is executed from cluster-api-provider-agent build and not assisted-service
This is what the job pass as INDEX_IMAGE:
[2022-09-19 19:44:54] ++(/root/config:6): source(): export INDEX_IMAGE=registry.build04.ci.openshift.org/ci-op-xxi8iphf/pipeline:ci-index

And this is what we have in the script that creates the catalog
```
  tee << EOCR >(oc apply -f -)
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: ${catalog_source}
  namespace: openshift-marketplace
spec:
  sourceType: grpc
  image: ${INDEX_IMAGE}
  displayName: Assisted Test Registry
  publisher: Assisted Developer
EOCR
  fi
```
So the images seems right, the problem is that this image is somewhat malformed/missing?
```
message: 'Failed to pull image "registry.build04.ci.openshift.org/ci-op-xxi8iphf/pipeline:ci-index":
rpc error: code = Unknown desc = reading manifest ci-index in registry.build04.ci.openshift.org/ci-op-xxi8iphf/pipeline:
manifest unknown: manifest unknown'
```
So I created this Pr to check what happens if I get the index image from 2.6, it seems that nothing changed :confused:

Reply:
I think this is the reason: https://github.com/openshift/release/pull/18771#discussion_r646074414
But why all of a sudden it's an issue? Didn't it pass in the past?

Reply:
fff :cry: 

> Please add documentation of why it is being done. Is there a ticket to solve it?